### PR TITLE
Fabo/fix participant images

### PR DESCRIPTION
**Pull request by faboweb**
Show fallback when no image available

### CODE REVIEW

**Review by jbibla — APPROVED** [review]

File: app/src/components/governance/ParticipantList.vue [hunk 1]
```diff
@@ -76,6 +76,10 @@ export default {
       type: Array,
       required: true,
     },
+    showAmounts: {
+      type: Boolean,
+      default: false
+    }
```
Comment (Bitcoinera) [inline app/src/components/governance/ParticipantList.vue:82]: I am just seeing in the `ParticipantList` code that the usage of `showAmounts` in quite counter-intuitive no?

Like it is exactly the opposite of what you would expect:

```
        <div v-if="!showAmounts && participant.option">
          <span class="option">{{ participant.option }}</span>
        </div>
        <div v-if="!showAmounts && participant.amount">
          <span class="amount">{{ participant.amount.amount }}</span>
          <span>{{ participant.amount.denom.concat(`s`) }}</span>
        </div>
```

We should definitely invert the values here

Comment (jbibla) [reply app/src/components/governance/ParticipantList.vue:82]: yeah - shouldn't it be:

`<div v-if="showAmounts && participant.amount">`

Comment (Bitcoinera) [reply app/src/components/governance/ParticipantList.vue:82]: OK, I am going to change it in this other PR #4939

Comment (faboweb) [reply app/src/components/governance/ParticipantList.vue:82]: `showAmounts` was there before and I think it referred to the amount of total stake?

Comment (Bitcoinera) [reply app/src/components/governance/ParticipantList.vue:82]: it refers whether or not the participants should display vote options & deposit amount. Yes, it was there and wrong from before

**Pull request merged**